scripts/scenarios/rally/rally-test.json

```diff
@@ -877,13 +877,13 @@
         {
             "args": {
                 "flavor": {
-                    "name": "m1.tiny"
+                    "name": "{{flavor_name}}"
                 },
                 "image": {
                     "name": "{{image_name}}"
                 },
                 "to_flavor": {
-                    "name": "{{flavor_name}}"
+                    "name": "m1.small"
```

Comment:
I do not get this, I think that the only effective change is the image name (cirros), because still 'flavor' is tiny, and 'to_flavor' is 'small'. Do you need to parametrize both?